doc/sphinx/addendum/extraction.rst

```diff
@@ -295,6 +295,28 @@ what ML term corresponds to a given axiom.
         Axiom Y : Set -> Set -> Set.
         Extract Constant Y "'a" "'b" => " 'a * 'b ".
 
+   .. note::
+      The extraction recognizes whether the realized axiom
+      should become a ML type constant or a ML object declaration. For example:
+
+      .. coqtop:: in
+
+         Axiom X:Set.
+         Axiom x:X.
+         Extract Constant X => "int".
+         Extract Constant x => "0".
+
+   .. caution:: It is the responsibility of the user to ensure that the ML
+      terms given to realize the axioms do have the expected types. In
+      fact, the strings containing realizing code are just copied to the
+      extracted files.
+
+   .. exn:: The term @qualid is already defined as foreign custom constant.
+
+      The :n:`@qualid` was previously used in a
+      :cmd:`Extract Foreign Constant` command. Using :cmd:`Extract Inlined Constant`
+      for :n:`@qualid` would override this command.
+
```

Comment:
Given the message, it  seems specific to `Extract Inlined Constant`, so it should be moved below.

Reply:
Indeed, that does make sense.

Reply:
Fixed.